mock_data

```diff
@@ -0,0 +1,205 @@
+var sys = require('sys')
+var fs = require('fs')
+var exec = require('child_process').exec;
+var program = require('commander');
+var databases = []
+program
+    .version('0.0.2')
+    .parse(process.argv);
+if (!program.args[0]) return console.log('No CouchDB URL provided. Quiting now.')
+var couchUrl = program.args[0]
+var nano = require('nano')(couchUrl)
+
+function start() {
+    increaseLimits()
+}
+
+// Increase Limits so couchapp push works correctly
+function increaseLimits() {
+    exec('sudo launchctl limit maxfiles 150056 150056', function doneWithLaunchCtl() {
+        exec('sudo ulimit -n 150056', function doneWithUlimit() {
+            return createDummyCourses()
+        })
+    })
+}
+
+var n_courses = 1 ; //Number of Iteration to run for courses 
```

Comment:
@dogi  as i have initialize variable var n_courses = 1 as a static value we can remove variable and loop statement for course if we need only one course.